sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerModelFactory.cs

```diff
@@ -175,9 +175,55 @@ public static FieldData FieldData(FieldBoundingBox boundingBox, int pageNumber,
         /// <summary>
         /// Initializes a new instance of the <see cref="FieldValue"/> structure.
         /// </summary>
-        /// <param name="value">The actual field value.</param>
+        /// <param name="type"></param>
+        /// <param name="text"></param>
+        /// <param name="valueString"></param>
+        /// <param name="valueDate"></param>
+        /// <param name="valueTime"></param>
+        /// <param name="valuePhoneNumber"></param>
+        /// <param name="valueNumber"></param>
+        /// <param name="valueInteger"></param>
+        /// <param name="valueSelectionMark"></param>
+        /// <param name="valueGender"></param>
+        /// <param name="valueCountry"></param>
         /// <returns>A new <see cref="FieldValue"/> instance for mocking.</returns>
-        public static FieldValue FieldValueWithStringValueType(string value) =>
+        public static FieldValue FieldValue(
+            FieldValueType type,
+            string text = default,
+            string valueString = default,
+            DateTimeOffset? valueDate = default,
+            TimeSpan? valueTime = default,
+            string valuePhoneNumber = default,
+            float? valueNumber = default,
+            long? valueInteger = default,
+            /*IReadOnlyList<FormField> valueArray,
+             * IReadOnlyDictionary<string, FormField> valueObject,*/
+            SelectionMarkState? valueSelectionMark = default,
+            FieldValueGender? valueGender = default,
+            string valueCountry = default)
+        {
+            return new FieldValue(
+                type,
+                valueString,
+                valueDate,
+                valueTime,
+                valuePhoneNumber,
+                valueNumber,
+                valueInteger,
+                default,
```

Comment:
At this point, I will need to implement a way to pass from `FormField` to `FieldValue_internal` so the user can use this mocking method.
But that is besides this effort, so will do later